README.md

```diff
@@ -2,14 +2,22 @@
 
 This repository is used to store documentation on NeoForge, the Minecraft modding API. It also contains documentation on NeoGradle, a Gradle plugin for developing NeoForge and mods using NeoForge.
 
-The documentation is built using [Docusaurus 2](https://docusaurus.io)
+The documentation is built using [Docusaurus 3](https://docusaurus.io)
 
 ## Contributing
 
 You can read the [contribution guidelines on the docs](https://docs.neoforged.net/contributing/).
 
 If you wish to contribute to the documentation, fork and clone this repository.
 
+The documentation uses Node.js 18. This can either be install manually or using a version manager that supports `.node-version` or `.nvmrc`. One or both of the following commands can be used to install and use the correct Node.js version:
```

Comment:
```suggestion
The documentation uses Node.js 18. This can either be installed manually or using a version manager that supports `.node-version` or `.nvmrc`. One or both of the following commands can be used to install and use the correct Node.js version:
```

Reply:
Whoops, grammar